Tests/pepdigest/broken_mainheader.pepdigest

```diff
@@ -0,0 +1,4 @@
+########################################
+# Program: pepdigest
+# Rundate: Sun 13 Jun 2021 13:18:29
+# Commandline: pepdigest
```

Comment:
Does this file (and the others) really not end with a new line character?

Reply:
In the `broken_*` files I didn't add the newline at the end since they are testing cases where the input file is not in the correct format, but I could add the newlines if you think it's usefull

Reply:
Ah, I missed the broken prefix. That should be fine as is then.